wee_slack.py

```diff
@@ -3021,6 +3042,8 @@ def subprocess_thread_message(message_json, eventrouter, team, channel, history_
 
             if parent_message.thread_channel and parent_message.thread_channel.active:
                 parent_message.thread_channel.buffer_prnt(message.sender, parent_message.thread_channel.render(message), message.ts, tag_nick=message.sender_plain)
+            elif message.ts > message.message_json.get("last_read", SlackTS()) and parent_message.subscribed:
```

Comment:
Shouldn't this be `message.ts > parent_message.message_json.get("last_read", SlackTS())`?